### DISCUSSION TITLE
Thanks for the Libre 3 patch.

### DISCUSSION THREAD

**Discussion by Lada-man** [maheini/FreeStyle-Libre-3-patch#33] [General]
Thanks for the Libre 3 patch. 
I have used Juggluco for 2 weeks and it works great, but it loose contact with my phone once or twice a day. I only have to place the phone close to the sensor to reconnect. This never happen when I used the Libre 3 App.
Same Phone, Samsung S20 and the same user behavior.
Any idea?

**Comment by maheini** [comment]
Hi @Lada-man 

Happy to hear about your success. Your problems might be caused by the latest Juggluco version (there where other reports like this as well). 
Please try those steps:

1. Update Juggluco to the newest version
2. Check if Battery optimization is set to manual / always running for Juggluco
3. Clear App Cache of Juggluco
4. Open Juggluco (may lead to crashes, that's normal)
5. Restart your sensor with Juggluco
6. Uninstall Libre 3 App (Just for testing purposes, to prevent any possible conflicts)

It may be helpful for others, if you can share your findings and mark this answer as Solution. Thank you 😄

**Reply by maheini** [reply]
Hi @Lada-man 
Any updates on this one?
